akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/Behaviors.scala

```diff
@@ -383,7 +383,7 @@ object Behaviors {
     }
 
     val mdcForMessageFun: T => Map[String, String] =
-      if (mdcForMessage == null) Map.empty
+      if (mdcForMessage == null) _ => Map.empty
```

Comment:
Curious that it wasn't caught by the compiler

Reply:
`Map.empty` have method `apply()` and compiler here assumes it have type of  `Map[T, Map[String, String]]`